torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1515,32 +1557,23 @@ void insertQuantDeQuantCall(
   } else {
     quantize_func = "quantize_per_tensor";
   }
+  Value* original_val = observer->input(1);
```

Comment:
could you add comments for `original_val` and `v`, it's starting to get a bit confusing